src/Omega_h_file.hpp

```diff
@@ -33,6 +33,8 @@ void write_sol(Mesh* mesh, std::string const& filepath,
 namespace meshsim {
 Mesh read(filesystem::path const& mesh, filesystem::path const& model,
           CommPtr comm);
+Mesh read(filesystem::path const& mesh, filesystem::path const& model,
```

Comment:
Let's add documentation on this function. At a minimum it needs to say that the numbering is a `.nex` file.

Reply:
Docs added with https://github.com/SCOREC/omega_h/pull/40/commits/1b89b5e560616b5ee7f09a875d78cf57324ca17c.